src/components/Column/ColumnSettings.tsx

```diff
@@ -36,109 +40,66 @@ export const ColumnSettings: FC<ColumnSettingsProps> = ({id, name, color, visibl
     dispatch(Actions.createColumnOptimistically({id: TEMPORARY_COLUMN_ID, name: "", color: randomColor, visible: false, index: columnIndex}));
   };
 
+  const renderColorPicker = () =>
+    openedColorPicker ? (
+      <ColorPicker id={id} name={name} visible={visible} index={index} color={color} onClose={onClose} colors={COLOR_ORDER} />
+    ) : (
+      <span className={`column__header-color-option ${color}_selected`} />
```

Comment:
`${color}_selected` doesn't look like valid BEM

I think something like `column__header-color-option--${color}-selected` would be valid